### ISSUE TITLE
Permission missing from the article

### ISSUE THREAD

**Issue by CiprianRistea** [microsoftgraph/microsoft-graph-docs-contrib#22]
The documentation should be updated to specify that the "Directory.AccessAsUser.All" permission should be used in the API call as well, otherwise a 403 error will be returned.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 154e8236-7344-04e4-561f-ff203e93d38f
* Version Independent ID: 2afc3fef-b3f3-28ff-938b-fbac56a02e34
* Content: [Create samlOrWsFedExternalDomainFederation - Microsoft Graph beta](https://learn.microsoft.com/en-us/graph/api/samlorwsfedexternaldomainfederation-post?view=graph-rest-beta&tabs=http)
* Content Source: [api-reference/beta/api/samlorwsfedexternaldomainfederation-post.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/api/samlorwsfedexternaldomainfederation-post.md)
* Product: **identity-and-sign-in**
* Technology: **microsoft-graph**
* GitHub Login: @namkedia
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by FaithOmbongi** [comment]
Hi @namkedia - any response for this claim?

**Comment by FaithOmbongi** [comment]
Closing this issue in readiness for deactivation of GitHub issues. See this post for more information https://github.com/microsoftgraph/microsoft-graph-docs-contrib/issues/9146.

I've filed internal bug 2858122 that's assigned to @namkedia to help investigate and resolve the issue. Permissions information are defined in an internal permissions model so the fix can't be made through a manual docs PR in the docs repo.
